### PR TITLE
update snapshot

### PR DESCRIPTION
**Pull request by Hailong-am**
### Description

update obsolete snapshot due to OUI upgrade 

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "@types/showdown": "^1.9.3",
     "enzyme-adapter-react-16": "^1.15.5",
     "jest-dom": "^4.0.0",
-    "cypress": "^6.0.0"
+    "cypress": "9.5.4"
```
Comment (SuZhou-Joe) [inline package.json:29]: Is this change needed?

Comment (Hailong-am) [reply package.json:29]: come from here https://github.com/opensearch-project/dashboards-notifications/actions/runs/8290160637/job/22687774853

Either we add `--single-version=loose/ignore` in the CI or upgrade to same version will resolve the CI failure.
As ftr repo using the cypress 9.5.4, so i decide to upgrade to same version to avoid any potential version issue when copy the test to ftr repo and fix the version conflict here.

```
ERROR [single_version_dependencies] Multiple version ranges for the same dependency
      were found declared across different package.json files. Please consolidate
      those to match across all package.json files. Different versions for the
      same dependency is not supported.

      If you have questions about this please reach out to the operations team.

      The conflicting dependencies are:

        cypress
          9.5.4 => opensearch-dashboards
          ^6.0.0 => notifications-dashboards
```

**Review by SuZhou-Joe — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (SuZhou-Joe) [reply package.json:29]: I see, thanks for the clarification.

**Review by zhichao-aws — APPROVED** [review]
